### ISSUE TITLE
build failed on loongarch64

### ISSUE THREAD

**Issue by zoze0** [nushell/nushell#11048]
### Describe the bug

`git clone, cargo build`  or  `cargo install nu` get the same error:
```rust
error[E0432]: unresolved import `crate::general`
  --> /home/loong/.cargo/bin/registry/src/rsproxy.cn-0dccff568467c15b/linux-raw-sys-0.1.4/src/lib.rs:84:16
   |
84 |     use crate::general::{cmsghdr, msghdr};
   |                ^^^^^^^ could not find `general` in the crate root

error[E0433]: failed to resolve: use of undeclared crate or module `general`
  --> /home/loong/.cargo/bin/registry/src/rsproxy.cn-0dccff568467c15b/linux-raw-sys-0.1.4/src/lib.rs:67:20
   |
67 | impl PartialEq for general::__kernel_timespec {
   |                    ^^^^^^^ use of undeclared crate or module `general`

error[E0433]: failed to resolve: use of undeclared crate or module `general`
  --> /home/loong/.cargo/bin/registry/src/rsproxy.cn-0dccff568467c15b/linux-raw-sys-0.1.4/src/lib.rs:79:13
   |
79 | impl Eq for general::__kernel_timespec {}
   |             ^^^^^^^ use of undeclared crate or module `general`

Some errors have detailed explanations: E0432, E0433.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `linux-raw-sys` (lib) due to 3 previous errors
warning: build failed, waiting for other jobs to finish...
```

Maybe it can be solved by upgrade `linux-raw-sys`,  
 [https://github.com/sunfishcode/linux-raw-sys/issues/34](https://github.com/sunfishcode/linux-raw-sys/issues/34) mentioned The Loongarch support starts from `linux-raw-sys v0.3.2`


### How to reproduce

`git clone, cargo build`  or  `cargo install nu`  on loongarch64 platform

### Expected behavior

could compile `linux-raw-sys` 

### Screenshots

_No response_

### Configuration

| key                | value                  |
| ------------------ | -----------------------|
| version            | 0.86.0         	      |
| build_os           | linux-loongarch64      |
| rust_version       | rustc 1.73.0 /1.71.0   |
| cargo_version      | cargo 1.73     	      |                                                                                                                        
| build_rust_channel | release       	      |                                                                                                                                                                                                                                      

### Additional context

_No response_

**Comment by zoze0** [comment]
PR #11115  fixed this issue by updating dependencies 

form
```rust
procfs v0.15.1
    └── rustix v0.36.17
            └── linux-raw-sys v0.1.4
```
to 
```rust
procfs v0.16.0
    └── rustix v0.38.21
            └── linux-raw-sys v0.4.10
```

**Comment by zoze0** [comment]
There is another issue (https://github.com/uutils/coreutils/issues/5572) reported on `uutils/coreutils v0.0.23`, We need to wait for it to release a new version or depend on main branch before we can build nushell successfully on LoongArch64

```c
~> version                                                                               
╭────────────────────┬──────────────────────────────────────────────────────────────────╮
│ version            │ 0.87.2                                                           │
│ branch             │ main                                                             │
│ commit_hash        │ e5d9f405dbdbb033f46cdae4a1832edd782b03fa                         │
│ build_os           │ linux-loongarch64                                                │
│ build_target       │ loongarch64-unknown-linux-gnu                                    │
│ rust_version       │ rustc 1.73.0 (cc66ad468 2023-10-03) (Arch Linux rust 1:1.73.0-1) │
│ cargo_version      │ cargo 1.73.0                                                     │
│ build_time         │ 2023-12-01 20:35:39 +08:00                                       │
│ build_rust_channel │ release                                                          │
│ allocator          │ mimalloc                                                         │
│ features           │ default, sqlite, trash, which, zip                               │
│ installed_plugins  │                                                                  │
╰────────────────────┴──────────────────────────────────────────────────────────────────╯
```

**Comment by zoze0** [comment]
```
╭────────────────────┬─────────────────────────────────────╮
│ version            │ 0.91.0                              │
│ branch             │                                     │
│ commit_hash        │                                     │
│ build_os           │ linux-loongarch64                   │
│ build_target       │ loongarch64-unknown-linux-gnu       │
│ rust_version       │ rustc 1.76.0 (07dca489a 2024-02-04) │
│ cargo_version      │ cargo 1.76.0 (c84b36747 2024-01-18) │
│ build_time         │ 2024-03-08 19:59:32 +08:00          │
│ build_rust_channel │ release                             │
│ allocator          │ mimalloc                            │
│ features           │ default, sqlite, trash, which, zip  │
│ installed_plugins  │                                     │
╰────────────────────┴─────────────────────────────────────╯
```
After the release of [wl-clipboard-rs](https://github.com/YaLTeR/wl-clipboard-rs/issues/41)  0.8.1 yesterday,  we can install nushell by `cargo install nu` . Everything built successfully.
@hustcer
